packages/entities/entities-shared/src/components/entity-link/EntityLink.vue

```diff
@@ -30,7 +31,7 @@
       </KTooltip>
     </KExternalLink>
     <KTooltip
-      v-if="entityUuid"
+      v-if="entityUuid && external"
```

Comment:
question: why is the tooltip only needed if an external link?

Reply:
We _should_ probably add a second prop that controls this separate from the `external` boolean. 

I'll check w/ UX on whether they see this as a potential use case (eg: we hide the external link icon, but then keep the "copy uuid or name" tooltip)

Reply:
@adamdehaven How about I default the Copy ID tooltip to `true`, and add a second `allowCopy` prop?  That way we can control the two icons independent of each other.

Reply:
TBH I'm not completely familiar with how this component is used specifically in all of the shared components. We'd need to check the usage there first

Reply:
Currently, it's only used in OverviewOnboardingCards' TopNTables in Gateway Manager:
https://github.com/search?q=repo%3AKong%2Fkonnect-ui-apps%20EntityLink&type=code

And used by API Requests slideout in Konnect:
https://github.com/search?q=repo%3AKong%2Fkhcp-ui%20%3CEntityLink&type=code

So it's safe to make this changes, adoption is not quite yet widespread.